### PR TITLE
Buffs nitrous oxide

### PR DESCRIPTION
**Pull request by swindly**
:cl: Swindly
balance: Nitrous oxide now causes anemia.
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) 
Let's see how many units of nitrous oxide have been made in [the](https://atlantaned.space/newSS13tools/stats/monthlyStats.php?year=2017&month=02&stat=chemical_reaction) [three](https://atlantaned.space/newSS13tools/stats/monthlyStats.php?year=2017&month=03&stat=chemical_reaction) [months](https://atlantaned.space/newSS13tools/stats/monthlyStats.php?year=2017&month=04&stat=chemical_reaction) it's been tracked:

48

Putting barriers of production on an explosive over which a strict improvement exists was a mistake. Giving it another use will give people reason to make it.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1122,7 +1122,19 @@
...
-
+/datum/reagent/nitrous_oxide/reaction_mob(mob/M, method=TOUCH, reac_volume)
+	if(method == VAPOR)
+		M.drowsyness += max(round(reac_volume, 1), 2)
+		
+/datum/reagent/nitrous_oxide/on_mob_life(mob/living/M)
+	M.drowsyness += 2
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.blood_volume -= 2.5
```
Comment (octareenroon91) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: What keeps blood_volume positive?

**Comment by FantasticFwoosh** [comment]
This makes No2 lethal to races that actually breath oxygen, meaning that people in operations where anesthetic No2 is used (circumstances like trying to operate on a traitor) will die, and losing significant amounts of blood will double up how fast you fall into air deprivation &

No2 currently in the game is a substitute knockout gas that *is* breathable, and this summarily nerfs using it in surgery (for reasons listed above) and also in the case of a spill, people are going to release No2, be knocked out and slowly and inevitably die while being completely unable to wake up.

> Please no thank you, this was not mechanically thought out enough to our uses in the game

**Comment by duncathan** [comment]
This is the reagent, not the gas @FantasticFwoosh

**Comment by banthebantz** [comment]
people never make this so I support

**Comment by kevinz000** [comment]
makei t deplete faster. this seems op.

**Comment by octareenroon91** [comment]
I'm not sure there's a way to survive a 100u dose

**Comment by swindly** [comment]
There isn't a way to survive a 100u dose of most chemicals.

**Comment by octareenroon91** [comment]
I mean a treatment.

I think a continuous infusion of charcoal, iron, and salglu might save the patient, but I'd have to test things.

**Comment by Iamgoofball** [comment]
>forgetting calomel

they always forget calomel

**Comment by ExcessiveUseOfCobblestone** [comment]
And penetic if you're feeling adventurous

**Pull request merged**